rental_class.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    @person.rentals = []
```

Comment:
- [ ] Like book, kindly ensure that the rental is adding itself to the @person achieving the 'belongs-to' association. e.g
`person.rentals << self`